app/config/Modules.scala

```diff
@@ -18,15 +18,16 @@ package config
 
 import controllers.actions.{AuthAction, AuthActionImpl, ServiceInfoAction, ServiceInfoActionImpl}
 import play.api.inject.{Binding, Module}
-import play.api.{Configuration, Environment}
-import uk.gov.hmrc.play.bootstrap.http.{DefaultHttpClient, HttpClient}
+import play.api.{Configuration, Environment, inject}
+import uk.gov.hmrc.http.HttpClient
+import uk.gov.hmrc.play.bootstrap.http.DefaultHttpClient
 
 class Modules extends Module {
 
   override def bindings(environment: Environment, configuration: Configuration): Seq[Binding[_]] = Seq(
-    bind(classOf[HttpClient]) to classOf[DefaultHttpClient],
-    bind(classOf[AuthAction]) to classOf[AuthActionImpl],
-    bind(classOf[ServiceInfoAction]) to classOf[ServiceInfoActionImpl]
+    inject.bind(classOf[HttpClient]) to classOf[DefaultHttpClient],
+    inject.bind(classOf[AuthAction]) to classOf[AuthActionImpl],
+    inject.bind(classOf[ServiceInfoAction]) to classOf[ServiceInfoActionImpl]
```

Comment:
Cleaner way:
    bind[HttpClient] to classOf[DefaultHttpClient],
    bind[AuthAction] to classOf[AuthActionImpl],
    bind[ServiceInfoAction] to classOf[ServiceInfoActionImpl]